src/definitions/modules/dropdown.js

```diff
@@ -345,11 +346,18 @@ $.fn.dropdown = function(parameters) {
             if( module.is.search() && !module.has.search() ) {
               module.verbose('Adding search input');
               $search = $('<input />')
-                .addClass(className.search)
-                .prop('autocomplete', 'off')
+                  .addClass(className.search)
+                  .prop({
+                      'autocomplete': 'off',
+                      'title': title
```

Comment:
As i guess the visual indicator using default browser label is too annoying, we should use the aria-namespace and change the `title` property to `aria-label`
![image](https://user-images.githubusercontent.com/18379884/58027217-71c9e880-7b18-11e9-87a9-bd906a02628a.png)